README.md

```diff
@@ -16,6 +16,17 @@ xattr -d com.apple.quarantine ./path/to/gof5_darwin
 
 On FreeBSD only `wireguard` or `ppp` (a wrapper around the ppp binary) drivers work. However the `wireguard` driver requires a [FreeBSD patch](freebsd.patch) applied before the compilation.
 
+## Linux
+
+Wireguard comes on most kernels, so you only need to install any dependencies, if you want to `ppp`
+
+### Arch Linux
+
+You can install the package directly from [AUR](https://aur.archlinux.org/packages/gof5/) based on the latast release, you can install with with AUR wrapper like:
```

Comment:
Can you add a primary note that latest releases are built using Travis CI and available in this repository releases section.

Reply:
Added, can you please review?